### PR TITLE
Update branch

### PR DESCRIPTION
**Pull request by soesandarwin2201**
## What does this PR do?
This PR is raised to use transactions to update and delete records

## Description of the task to be completed
- [x] Add a column species of type string to your animals table
- [x] Insert the  data
- [x] -Inside a transaction:
- [x] Delete all animals born after Jan 1st, 2022.
- [x] Create a savepoint for the transaction.
- [x] Update all animals' weight to be their weight multiplied by -1.
- [x] Rollback to the savepoint
- [x] Update all animals' weights that are negative to be their weight multiplied by -1.
- [x] Commit transaction

## How should it be tested
Clone the repository
```
git clone  https://github.com/soesandarwin2201/vet_clinic.git
```
Navigate to the folder
```
cd vet_clinic
```
Checkout the branch
```
git checkout update-branch

```
Screenshot for database 

Screenshot (334)

Screenshot (335)

Screenshot (336)
Screenshot (337)
Screenshot (338)

Modify methods
Screenshot (339)

### CODE REVIEW

File: queries.sql [hunk 1]
```diff
@@ -8,3 +8,30 @@ SELECT name,escape_attempts FROM animals WHERE weight_kg > 10.5;
...
+UPDATE animals SET weight_kg = weight_kg * -1;
+ROLLBACK TO delete_date;
+UPDATE animals SET weight_kg = weight_kg * -1 WHERE weight_kg < 0;
+COMMIT;
+
+SELECT COUNT(*) FROM animals;
+SELECT COUNT(*) FROM animals WHERE escape_attempts = 0;
+SELECT AVG(weight_kg) FROM animals;
+SELECT MAX(escape_attempts) FROM animals WHERE neutered = true OR neutered = false;
+SELECT MIN(weight_kg),MAX(weight_kg) FROM animals;
```
Comment (gemmen29) [inline queries.sql]: - [x] Kindly fix these three select statements to match the requirement:
   - `Who escapes the most, neutered or not neutered animals?`.
   - `What is the minimum and maximum weight of each type of animal?`.
   - `What is the average number of escape attempts per animal type of those born between 1990 and 2000?`

### Hint: 
```
use the 'GROUP BY' clause
```

**Review by gemmen29 — CHANGES_REQUESTED** [review]
Hi @soesandarwin2201, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights 💯 
- Descriptive pull request ✔️ 
- Descriptive commit messages ✔️ 

## Required Changes ♻️
_Check the comments under the review._



## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: queries.sql (hunk 1 above)
Comment (soesandarwin2201) [reply queries.sql]: Thanks for pointing out to improve my code , It was helpful to learn.Thanks for your time and efforts

**Review by gemmen29 — APPROVED** [review]
Hi @soesandarwin2201, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights 💯 
- Descriptive pull request ✔️ 
- Descriptive commit messages ✔️ 



## Optional suggestions

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**